test/ci.test.js

```diff
@@ -41,3 +41,15 @@ test("is CI required false", async () => {
   // Assert
   expect(!result);
 });
+
+test("isCIRequired paths file not defined", async () => {
+  //Arrange
+  const context = {
+    config: jest.fn(() => null)
```

Comment:
sorry for being so strict here, but in case `bot-files/paths.yml` is not present, `triggerPaths` will be undefined so I would replace `null` by `undefined` as I suggested in my previous review.
https://stackoverflow.com/a/5076962/2358463

Reply:
Ah okay